pysatNASA/instruments/de2_rpa.py

```diff
@@ -26,7 +26,7 @@
 spectrum; and the concentration of H+, He+, O+, and Fe+, and of molecular ions
 near perigee.
 
-It includes the DUCT portion of the high resolutiondata from the Dynamics
+It includes the DUCT portion of the high resolution data from the Dynamics
```

Comment:
what does DUCT mean? When I tried looking this up all I found was that this RPA has "a separate wide aperture sensor, a duct sensor, was flown"

Reply:
I'm not sure if it's an acronym or just a name that's been capitalized (like MAG).  I might need to drop this info since we don't include the duct files, which require an xarray load.

Reply:
Removing these lines since duct is not yet supported